src/components/pages/discovery/DiscoveryContainer.js

```diff
@@ -41,8 +41,42 @@ export const mapStateToProps = (state, ownProps) => {
   }
 }
 
+function are_valid_coordinates(coordinates) {
+  return (
+    coordinates !== null &&
+    coordinates !== undefined &&
+    coordinates.latitude !== null &&
+    coordinates.longitude !== null
+  )
```

Comment:
```suggestion
  return (
    coordinates && coordinates.latitude && coordinates.longitude
  )
```
ça fonctionne ça par hasard ?